### DISCUSSION TITLE
「サイト名」未入力時に「Connect-CMS」と表示せず、空白とする提案

### DISCUSSION THREAD

**Discussion by ghost** [opensource-workshop/connect-cms#1308] [Ideas]
<img width="1353" alt="ふちゅうテクノフェアdemo" src="https://user-images.githubusercontent.com/107566285/173864458-d8eeea19-9fb9-46ea-951f-a05906930300.png">
「管理者メニュー」→「サイト管理」→「サイト名」を空欄にしたところ、コンテンツ画面のヘッダー(?)部に「Connect-CMS」と表示されてしまいました。空白にしたいのですが、ムリでしょうか。（上の写真のように今は「ふちゅうテクノフェア2022」と表示させています。Connect-CMSと表示させるわけにもいかず。。。）

本来は、単にヘッダー部を非表示にしたかっただけなのですが、そうするとスマホ画面ではハンバーガーメニューが表示されません。ハンバーガーメニューは必要ですので、せめてヘッダー部は「サイト名」表示無しで余白として使いたいと考えております。

いかがでしょう？

**Comment by gakigaki** [comment]
ヘッダー部のサイト名を消すのは、以下の対応で可能になります。

- 「管理者メニュー」→「サイト管理」→「サイト名」を空白にする
- .envファイルのAPP_NAMEをConnect-CMSから空白にする

また、スマホのハンバーガーメニューですが、
メニュープラグインにハンバーガーメニューのテンプレートが数種類あります。
ヘッダーを非表示にさせて、メニュープラグインを使う方法もあるので、
そちらも併せてご確認いただければと思います。

**Reply by ghost** [reply]
ありがとうございます。それら方法を試してみます。

**Comment by akagane99** [comment]
過去の記事で、cssで消せる方法がありました。
参考まで。
* https://github.com/opensource-workshop/connect-cms/issues/811#issuecomment-841825229

**Reply by ghost** [reply]
ありがとうございます。
ちょっと強引なCSSですね(^^;;

> ページのtitleタグでサイト名使われてるので、サイト名空だとGoogleとか検索エンジンからの表示がどうなっちゃうかな？と思いました。

こちらの指摘が気になりますね。titleタグに使われているのであれば、「サイト名」を空にするのはよくないですね。。。文字色を背景色と同じにして見えなくするのがいいかも。スマホではヘッダー部にサイト名が見えた方がよさそうですし、CSSでうまいこと対応するのが吉と思えてきました。
